playbooks/graphql-schema-design.md

```diff
@@ -123,6 +135,28 @@ e.g. an `artworks` field _and_ an `artworks_connection`, simply choose one form
 - In rare cases an immediate list may be used, but this should only be done in cases where the list has few entries
   and doesn’t require pagination, i.e. fetch _all_ entries at once.
 
+## ID fields
+
+An ID field that refers to e.g. a database ID should be called something like `internalID`, it MUST _never_ be called
+just `id`, as that name is reserved for ‘Global Object Identification’ (further explained in the next section).
+
+Its type should be `ID!`, which is a custom string scalar meant to convey that the value is an identifier and is not
+nullable, as database IDs never are `null`.
+
+## Global Object Identification
+
+All GraphQL services should follow the [Global Object Identification specification](https://facebook.github.io/relay/graphql/objectidentification.htm). Due to schema stitching, for their IDs services should encode their own service ID for metaphysics to be able to resolve a node ID back to its upstream service.
```

Comment:
Good call, esp with the Postgres primary keys, there would def be collisions (less common w/ the Mongo timestamped ones) so including a type and service is 👍